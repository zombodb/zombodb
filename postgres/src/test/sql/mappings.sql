----------
-- DATA --
----------

-- _meta --
SELECT zdb_get_index_mapping('so_posts')->'mappings'->'data'->'_meta'->>'primary_key' = 'id';

-- _all --
SELECT zdb_get_index_mapping('so_posts')->'mappings'->'data'->'_all'->>'analyzer' = 'phrase';

-- _source --
SELECT zdb_get_index_mapping('so_posts')->'mappings'->'data'->'_source'->>'enabled' = 'false';
