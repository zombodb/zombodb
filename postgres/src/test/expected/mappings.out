----------
-- DATA --
----------
-- _meta --
SELECT zdb_get_index_mapping('so_posts')->'mappings'->'data'->'_meta'->>'primary_key' = 'id';
 ?column? 
----------
 t
(1 row)

-- _all --
SELECT zdb_get_index_mapping('so_posts')->'mappings'->'data'->'_all'->>'analyzer' = 'phrase';
 ?column? 
----------
 t
(1 row)

-- _source --
SELECT zdb_get_index_mapping('so_posts')->'mappings'->'data'->'_source'->>'enabled' = 'false';
 ?column? 
----------
 t
(1 row)

